src/patternfly/components/List/list.scss

```diff
@@ -8,20 +8,20 @@
   --pf-c-list--m-grid--GridGap: var(--pf-global--spacer--md);
 
   // Content variables (no modifier)
+  --pf-c-content--MarginBottom: var(--pf-global--spacer--md);
```

Comment:
I'm curious if you can see any advantage to these vars being namespaced as `--pf-c-content`? shouldn't they just be `--pf-c-list`?